src/Analyzers/CSharp/CodeFixes/UsePatternCombinators/CSharpUsePatternCombinatorsCodeFixProvider.cs

```diff
@@ -118,7 +118,7 @@ private static PatternSyntax AsPatternSyntax(AnalyzedPattern pattern)
         private static ExpressionSyntax AsExpressionSyntax(ExpressionSyntax expr, AnalyzedPattern p)
         {
             var semanticModel = p.Target.SemanticModel;
-            Debug.Assert(semanticModel != null);
+            RoslynDebug.Assert(semanticModel != null);
```

Comment:
`Debug.Assert` isn't annotated for netstandard2.0. Used RoslynDebug instead of null-suppression with `!`.